Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainViewModel.swift

```diff
@@ -159,9 +166,16 @@ final class PlaygroundMainViewModel: ObservableObject {
                     case .completed(let session):
                         let accounts = session.accounts.data.filter { $0.last4 != nil }
                         let accountInfos = accounts.map { "\($0.institutionName) ....\($0.last4!)" }
+                        let sessionInfo =
+"""
+session_id=\(session.id)
```

Comment:
can you add indents?

Reply:
This is just an unfortunate combination of Xcode formatting for this particular syntax and our linter. It just automatically breaks it like this. Structuring it differently will either be impossible (due to linter), or highly inconvenient (due to Xcode automating to this formatting [you can press CTRL-I in Xcode to auto-format). In other words, I don't have an opinion on this formatting, it's simply a combination of Xcode defaults and our linter. 

Note that everything in "FinancialConnections Example" is by-definition code that is a bit more 'loose' in its implementation as its quick code to achieve easier testing. For example, _I could_ move this text to a function to make it a bit more prettier, but its over-engineering for the context.